docs/tox.ini

```diff
@@ -18,19 +18,22 @@ commands =
 
 [testenv:docs]
 description = Build the Sphinx docs
+allowlist_externals = rm
 deps = pip-tools
 commands_pre =
     pip-sync {toxinidir}/requirements.txt
 commands =
-    sphinx-build -W --keep-going . _build/html
+    rm -rf __pycache__ _build/
+    sphinx-build -W --keep-going -b dirhtml . _build/html
+    python -m http.server -b 127.0.0.1 -d _build/html/
```

Comment:
I think this block should just stay the "build" part, and the serving part should be in the "serve" Tox command. Or is this just for debugging?

Reply:
When I check out this code locally, run `tox -e docs`, then make changes to `openapi.yaml`, refreshing the web page doesn't update. I have to Ctrl-C to quit and then restart and then do a hard refresh in the browser. Is this expected? Can we make this a bit smoother for local development?